Request to merge

Koushith opened the pull request:
This Branch includes :
 - Home, About, Team, and Contact pages.
 - Added React Router
 - Replaced placeholder image with actual illustrations. 
 - Tablet Responsiveness.

Upcoming Changes : 
- [ ] Color schemes, mobile responsiveness,  and making pixel perfect according to the mockup.

![2020-07-28 11-30-24](https://user-images.githubusercontent.com/30016242/88628967-4d7a7c00-d0cc-11ea-949e-5ac337b18e28.gif)

Koushith committed:
added Router and fixed nav

Koushith committed:
added about page

Koushith committed:
minor Fixes

Koushith committed:
updated Home and other pages

Koushith committed:
Added general layout

abhinav-anshul commented:
-  Replace `class` with `className` in all the files as we are using `JSX` 
-  Maintain consistent naming convention for files, In the About folder, There is AboutUs.js as well as About_description, Therefore I would suggest either use `camel case` or `snake case`.

koshikraj commented on tailblocks/src/App.js:
```diff
@@ -1,22 +1,34 @@
+          <Route path='/team'>
+            <Team />
+          </Route>
+          <Route path='/contact'>
```
I think we should add an error page too. Right now it just shows an empty component.

Koushith replied:
Done. added 404 error page.

koshikraj commented on tailblocks/src/Pages/About.js:
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Header from '../components/Header';
+import Description from '../components/Description';
+import AboutDescription from '../components/About/About_description';
```
Let's use a consistent naming convention (`PascalCase` for the component files)

koshikraj commented on tailblocks/src/components/Teams/Members.js:
```diff
@@ -0,0 +1,124 @@
+                class='flex-shrink-0 rounded-lg w-full h-56 object-cover object-center mb-4'
+                src='https://dummyimage.com/203x203'
+              />
+              <div class='w-full'>
```
Shall we maintain the team details in a separate config file and import and iterate it here just to reduce the file size.

Koushith replied:
created a new config file that contains an array of objects and mapped in the Team section.

koshikraj commented:
Overall a pretty great PR as there are a lot of updates here :+1:. Good PR description too. Make sure to name the branch and commit messages according to our naming conventions in the future.

Koushith committed:
replaced class with className

Koushith committed:
replaced class with className and added 404 page

Koushith committed:
modified class with className, added 404 paage

Koushith committed:
adds 404 page,modified class with className and minor ehancement

Koushith commented:
Changes include  : 
- Added 404 error page.
- Maintained the team details in a separate config file and imported it on the Team page and mapped accordingly. 
- corrected the naming conventions.
- Fixed the bottom Navigation section.

abhinav-anshul reviewed (approved):
LGTM, Merging

abhinav-anshul committed:
Merge branch 'master' into Other-pages

koshikraj commented on tailblocks/src/App.js:
```diff
@@ -1,21 +1,37 @@
-import Technology from './components/Technology'
-import Footer from "./components/Footer"
+import React from 'react';
+import Home from './pages/home';
```
Looks like this path doesn't exist anymore. Check the case in each import. It causes issues on Mac and Linux.

Koushith replied:
Removed all unnecessary imports.

koshikraj commented on tailblocks/src/components/About/AboutUs.js:
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import About from '../../pages/about';
```
Looks like this path doesn't exist anymore. Check the case in each import. It causes issues on Mac and Linux.

Koushith replied:
Removed all unnecessary imports.

Koushith committed:
App.js

incorporates the review comment change requests

koshikraj committed:
fixes import module names
